### PR TITLE
Updating project voting members for Cross Project Council

### PR DESCRIPTION
**Pull request by joesepi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,21 +75,18 @@ CPC members should attend as many meetings as possible, and non-members are welc
...
+According to the [CPC Charter](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#voting-members), each of the Growth, At Large, and Incubating Projects may nominate a candidate to fill one of two voting seats on the CPC which represent this group of projects as a whole. From those nominees, two voting members are chosen via an [election process outlined in the CPC Governance](https://github.com/openjs-foundation/cross-project-council/blob/master/GOVERNANCE.md#election-process-for-non-impact-voting-cpc-members).
 
-- **Architect**: Kris Borchers (@kborchers)
-- **messageformat**: Eemeli Aro (@eemeli)
-- **Mocha**: Christopher Hiller (@boneskull)
-- **Node-RED**: Nick O'Leary (@knolleary)
-- **webhint**: Antón Molleda (@molant)
+- **Mocha**: Christopher Hiller (@boneskull, IBM)
+- **AMP**: Tobie Langel (@tobie, UnlockOpen)
```
Comment (tobie) [inline README.md:89]:
```suggestion
- Christopher Hiller (@boneskull, IBM)
- Tobie Langel (@tobie, UnlockOpen)
```

As we're representing the whole group of projects, I feel like the project affiliation isn't necessary.

Comment (kborchers) [reply README.md:89]: I agree. It is important to know affiliation during the election but after that it is not necessarily important to track that affiliation.

Comment (joesepi) [reply README.md:89]: I concur. Suggestion committed and appreciated.

**Comment by brianwarner** [comment]
Thanks, @knolleary @kborchers @molant @eemeli @dpgraham and @boneskull! 

I'll remove you from the cpc-voting-members team, and add @imurchie and @sokra.

**Comment by boneskull** [comment]
Errr, what?

**Comment by boneskull** [comment]
@brianwarner I am a voting member :smile:

**Comment by brianwarner** [comment]
Oops, sorry, yes - I was reading the commit directly :-) You're still there.

**Comment by boneskull** [comment]
@brianwarner I did get an email about being removed

**Comment by brianwarner** [comment]
Whoops. Sorry about that, fixed now.

**Pull request merged**